docs/post-call-survey.md

```diff
@@ -2,6 +2,41 @@
 
 You can configure post-call surveys to collect valuable feedback like quality of services or net promoter score after a call ends.
 
+## Create a 1-question poll
+
+### Configure using environment variables
+
+```
+VV_POSTCALL_SURVEY_TYPE = "onequestionpoll"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE = "<your_oneQuestionPollSurvey_title>"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT = "<your_oneQuestionPollSurvey_prompt>"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE = "<likeOrDislike | rating | text>"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_ANSWER_PLACEHOLDER = "<your_oneQuestionPollSurvey_answerPlaceholder_TextInput>"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT = "<your_oneQUestionPollSurvey_SaveButtonText>"
```

Comment:
We can be more specific here:

VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE = "Tell us how we did"
VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT = "How satisfied are you with this virtual appointment's audio and video quality?"
VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_ANSWER_PLACEHOLDER = "Tell us why"
VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT = "Continue"